nilearn/datasets/tests/test_utils.py

```diff
@@ -422,35 +422,57 @@ def test_safe_extract(tmp_path):
         _utils.uncompress_file(ztemp, verbose=0)
 
 
-def test_fetch_file_part(tmp_path):
+def test_fetch_single_file_part(tmp_path, capsys, request_mocker):
```

Comment:
Looks like `fetch_single_file` used to be called `fetch_file`, in which case the tests should be renamed too